test/test_code_utils.py

```diff
@@ -574,21 +574,29 @@ def test_get_powershell_command_pwsh(self, mock_subprocess_run):
 
         self.assertEqual(get_powershell_command(), "pwsh")
 
+        # mock_subprocess_run.side_effect = [FileNotFoundError, MagicMock(return_value=MagicMock(returncode=0, stdout=StringIO("7")))]
```

Comment:
Let's remove the outdated code (comments) here.

Reply:
Done!